tardis/scripts/tardis

```diff
@@ -1,13 +1,25 @@
 #!/usr/bin/env python
```

Comment:
I don't get it how it is related to pip install? 

It seems like it makes tardis runnable via CLI - is that so?

Reply:
**Some context:**

We had two `scripts` folder before APE 17 migration: 

- the`scripts/` (containing just the `tardis` UNIX script) 
- and `tardis/scripts` (Python entry points)

In #1674 made some changes to the `tardis` script to use it as an entry point. But seems that worked for `python setup.py develop`, but didn't worked for `pip install .` (required by the `conda-forge` packaging system, [see here](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=347242&view=logs&j=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&t=841356e0-85bb-57d8-dbbc-852e683d1642&l=691)).

I reverted those changes plus some minor changes to make the script tidier.

You can try the result of `pip install .` on this branch and compare it with `master`.